admin/onpage/class-ryte-service.php

```diff
@@ -0,0 +1,97 @@
+<?php
+/**
+ * @package WPSEO\Admin\OnPage
+ */
+
+/**
+ * Class WPSEO_OnPage_Service
+ */
+class WPSEO_Ryte_Service {
+
+	/**
+	 * @var WPSEO_OnPage_Option
+	 */
+	protected $onpage_option;
```

Comment:
With the future in mind where we possible rename the `OnPage_Option` to `Ryte_Option` it is possibly better to name the property `$option` instead of `$onpage_option`